packages/web-components/src/avatar/avatar.template.ts

```diff
@@ -21,10 +21,8 @@ const defaultIconTemplate = html`<svg
  */
 export function avatarTemplate<T extends Avatar>(): ElementViewTemplate<T> {
   return html<T>`
-    <template role="img" data-color=${x => x.generateColor()}>
-      <slot>${x => (x.name || x.initials ? x.generateInitials() : defaultIconTemplate)}</slot>
-      <slot name="badge"></slot>
-    </template>
+    <slot>${x => (x.name || x.initials ? x.generateInitials() : defaultIconTemplate)}</slot>
+    <slot name="badge"></slot>
   `;
 }
```

Comment:
Could this be:
```ts
const template = html<Avatar>`
    <slot>${x => (x.name || x.initials ? x.generateInitials() : defaultIconTemplate)}</slot>
    <slot name="badge"></slot>
`;
```

Reply:
Yes, I think we need to do a run through once we get things settled to determine what should be a function we export and what can be a static template - I'd like to do a holistic pass on that all at once though.